app.js

```diff
@@ -0,0 +1,18 @@
+window.addEventListener('load', () => {
+  const handleMenuBar = () => {
+    document.getElementById('mobile-nav').style.display = 'flex';
+    const elementsToHide = Array.from(document.querySelectorAll('.hide-nav-menu-mobile'));
+    elementsToHide.forEach((x) => { x.style.display = 'none'; });
+  };
+
+  const handleReverse = () => {
+    document.getElementById('mobile-nav').style.display = 'none';
+    const elementsToHide = Array.from(document.querySelectorAll('.hide-nav-menu-mobile'));
+    elementsToHide.forEach((x) => { x.style.display = 'flex'; });
+  };
+
+  document.getElementById('menu-btn').addEventListener('click', handleMenuBar);
+  document.getElementById('nav-btn1').addEventListener('click', handleReverse);
+  document.getElementById('nav-btn2').addEventListener('click', handleReverse);
+  document.getElementById('exit-btn').addEventListener('click', handleReverse);
+});
```

Comment:
- [ ] Kindly add an event listener for the contact link too. One of the project requirements specifies that  "When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears". Please see the snapshot below 
![Inkedmobile_menu_LI](https://user-images.githubusercontent.com/46851023/131741712-0cb02e6f-4820-472a-b2ba-b7af53be8076.jpg)

Reply:
- [ ] Kindly add an event listener for the contact option so it can display the contact section. One of the project requirements specifies that  "When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed".